src/utils.js

```diff
@@ -69,6 +69,21 @@ export const setQueryParam = (param, value) => {
   localStorage.savedQueryParams = qs.toString()
 }
 
+export const appendQueryParamElement = (param, value) => {
+  const qs = new URLSearchParams(window.location.search)
+  setQueryParam(param, qs.get(param) ?
+    [...new Set([...qs.get(param).split(','), value])].join(',') :
+    value)
```

Comment:
```suggestion
  setQueryParam(param, [...new Set(qs.get(param)?.split(',') ?? []).add(value)].join(',')
```

Reply:
Kind of annoying that this does not work (leaves a comma in front) as the optional chaining operator only cares about `null` and `undefined`, but not empty string...

Reply:
ahhh split changes `''` to `['']`, rip

Reply:
how about

```javascript
setQueryParam(param, [...new Set((qs.get(param) || null)?.split(',')).add(value)].join(',')
```

this doesn't matter at all lol but I like playing with the new js features

Reply:
Yep made that change and I think it works